<launch>
  <!-- Ejecutar los nodos necesarios con xterm -->
  <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" output="screen"/>
  <node pkg="beginner_tutorials" type="turtle_teleop_key" name="turtle_teleop_key" output="screen"/>

  <!-- Lanzar el nodo distance_checker_plus -->
  <node pkg="beginner_tutorials" type="distance_checker_plus" name="distance_checker_plus" output="screen"/>

  <!-- Ejecutar turtle_teleop_key con remap para la segunda tortuga -->
  <node pkg="beginner_tutorials" type="turtle_teleop_key" name="turtle_teleop_key22" output="screen">
    <remap from="/turtle1/cmd_vel" to="/turtle2/cmd_vel"/>
  </node>

  <!-- Lanzar el nodo distance_observer -->
  <node pkg="beginner_tutorials" type="distance_observer" name="distance_observer" output="screen"/>
</launch>
